cdap-docs/reference-manual/source/http-restful-api/logging.rst

```diff
@@ -13,7 +13,7 @@ Logging HTTP RESTful API
 
 Downloading Logs
 ----------------
-Logs emitted by any of the *Flows*, *Procedures*, *MapReduce* jobs, or
+Logs emitted by any of the *Flows*, *Procedures*, *MapReduce* programs, or
```

Comment:
Procedures aren't coming our quite yet, and adding Spark would be a different change. I will check on if Spark needs adding.